hummingbot/client/command/gateway_command.py

```diff
@@ -54,12 +54,12 @@ def gateway(self,
     async def _test_connection(self):
         # test that the gateway is running
         try:
-            resp = await self._api_request("get", "", {})
+            resp = await self._api_request("get", "", {}, True)
```

Comment:
again, just to improve the code readability, pass in the whole "fail_silently = True" keyword argument, rather than just "True".